logic/src/appleMain/kotlin/com/wire/kalium/logic/CoreLogic.kt

```diff
@@ -45,8 +50,17 @@ actual class CoreLogic(
             shouldEncryptData = kaliumConfigs.shouldEncryptData
         )
 
-    override val globalDatabase: GlobalDatabaseProvider =
-        GlobalDatabaseProvider("$rootPath/global-storage")
+    override val globalDatabase: GlobalDatabaseBuilder = globalDatabaseProvider(
```

Comment:
```suggestion
    override val globalDatabaseBuilder: GlobalDatabaseBuilder = globalDatabaseProvider(
```

Reply:
Applied in second PR, at:
https://github.com/wireapp/kalium/pull/2751/commits/1d75080ab221e5c57a0824b742320e49ca03cfa7